integration-tests/infra_deployments/deployer_test.go

```diff
@@ -31,21 +31,21 @@ var (
 			`
 )
 
-func createKeys(testState *testing.T) ([]*client.Chainlink, error) {
+func createKeys(testState *testing.T) ([]*client.ChainlinkK8sClient, error) {
 	urls := [][]string{
 		// Node access params
 		{"NODE_URL", "NODE_USER", "NODE_PASS"},
 	}
-	var clients []*client.Chainlink
+	var clients []*client.ChainlinkK8sClient
 
 	for _, nodeUrl := range urls {
 		u, _ := url.Parse(nodeUrl[0])
-		c, err := client.NewChainlink(&client.ChainlinkConfig{
+		c, err := client.NewChainlinkK8sClient(&client.ChainlinkConfig{
 			URL:        nodeUrl[0],
 			Email:      nodeUrl[1],
 			Password:   nodeUrl[2],
 			InternalIP: u.Host,
-		})
+		}, "", "")
```

Comment:
@smickovskid is this ok to leave empty?

Reply:
Yes, I will perhaps even delete this in the future, this was a workaround so we can deploy on an already deployed cluster. It can be like this for now